pages/glossary/explicit-section-of-content.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Explicit section of content
+key: explicit-section-of-content
+unambiguous: true
+objective: true
+---
+
+An _explicit section of content_ is a subtree of the [flat tree][], rooted at an element with a [semantic role][] which is one of the sectioning roles (defined just after), except if it is the only child of an element whose [semantic role][] is one of the sectioning roles.
+
+The sectioning roles considered here are: the [`document`][document] role; any role that inherits from [`document`][document]; any role that inherits from [`landmark`][landmark].
+
+If a descendant of the root of an explicit section of content is a [browsing context container][], then the full content of the associated [nested browsing context][] is also included in this explicit section of content.
+
+Explicit sections of content may have zero or one heading [associated][] with them.
+
+**Note:** The `article` role (inheriting from `document`) is included in this definition because the `article` element is [sectioning content][] used by the [algorithm for creating an outline][outline algorithm].
+
+**Note:** The roles considered here are close but different from the [implicit roles][] of [sectioning content][] and [sectioning root][] elements. Notably, the `body` element itself does not define an explicit section of content.
+
+**Note:** When one of these elements is the single child of a similar element, they form a single explicit section of content, rather than one each, because they have the exact same content.
```

Comment:
According to the definition, this happens not because they have the same content, but because the child node is not a section of content (because it is the single child of an element with sectioning role)

Reply:
Yes. This note is here to explain why a single child is ignored, I.e. "it's OK to ignore the single child of an element with sectioning role because otherwise there would be two sections with the exact same content and that wouldn't make sense".

I tried to turn the note around to make that intention clearer.